src/SprykerSdk/Zed/AopSdk/Business/OpenApi/Builder/OpenApiCodeBuilder.php

```diff
@@ -0,0 +1,732 @@
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AopSdk\Business\OpenApi\Builder;
+
+use cebe\openapi\Reader;
+use cebe\openapi\spec\OpenApi;
+use cebe\openapi\spec\Operation;
+use cebe\openapi\spec\PathItem;
+use cebe\openapi\spec\Reference;
+use cebe\openapi\spec\Schema;
+use Doctrine\Inflector\Inflector;
+use Generated\Shared\Transfer\MessageTransfer;
+use Generated\Shared\Transfer\OpenApiRequestTransfer;
+use Generated\Shared\Transfer\OpenApiResponseTransfer;
+use SprykerSdk\Zed\AopSdk\AopSdkConfig;
+use Symfony\Component\Process\Process;
+
+class OpenApiCodeBuilder implements OpenApiCodeBuilderInterface
+{
+    /**
+     * @var string
+     */
+    protected const SPRYKER = 'Spryker';
+
+    /**
+     * @var string
+     */
+    protected const CORE = 'core';
+
+    /**
+     * @var \SprykerSdk\Zed\AopSdk\AopSdkConfig
+     */
+    protected AopSdkConfig $config;
+
+    /**
+     * @var \Generated\Shared\Transfer\OpenApiResponseTransfer
+     */
+    protected OpenApiResponseTransfer $openApiResponseTransfer;
+
+    /**
+     * @var \Doctrine\Inflector\Inflector
+     */
+    protected Inflector $inflector;
+
+    /**
+     * @var string
+     */
+    protected string $sprykMode = 'project';
+
+    /**
+     * @var array
+     */
+    protected array $parseProperties = [];
+
+    /**
+     * @param \SprykerSdk\Zed\AopSdk\AopSdkConfig $config
+     * @param \Doctrine\Inflector\Inflector $inflector
+     */
+    public function __construct(AopSdkConfig $config, Inflector $inflector)
+    {
+        $this->config = $config;
+        $this->inflector = $inflector;
+        $this->openApiResponseTransfer = new OpenApiResponseTransfer();
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\OpenApiRequestTransfer $openApiRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\OpenApiResponseTransfer
+     */
+    public function build(OpenApiRequestTransfer $openApiRequestTransfer): OpenApiResponseTransfer
+    {
+        $openApi = $this->load($openApiRequestTransfer->getTargetFileOrFail());
+
+        $organization = $openApiRequestTransfer->getOrganizationOrFail();
+        if ($organization === static::SPRYKER) {
+            $this->sprykMode = static::CORE; //Set sprykMode based on organization
+        }
+
+        $this->generateTransfers($organization, $openApi);
+
+        return $this->openApiResponseTransfer;
+    }
+
+    /**
+     * @param string $openApiFilePath
+     *
+     * @return \cebe\openapi\spec\OpenApi
+     */
+    public function load(string $openApiFilePath): OpenApi
+    {
+        return Reader::readFromYamlFile((string)realpath($openApiFilePath));
+    }
+
+    /**
+     * @param string $path
+     * @param \cebe\openapi\spec\Operation $operation
+     *
+     * @return string
+     */
+    protected function getModuleName(string $path, Operation $operation): string
```

Comment:
No call for this method in the code above. It should be moved below the first method which calls it.